<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="spectogramLabel">
      <property name="minimumSize">
       <size>
        <width>430</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="touchStateLabel">
      <property name="minimumSize">
       <size>
        <width>430</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>430</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="cameraView">
      <property name="minimumSize">
       <size>
        <width>430</width>
        <height>340</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>430</width>
        <height>340</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>430</width>
        <height>340</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
